stix2/STIXPatternVisitor.py

```diff
@@ -0,0 +1,353 @@
+import importlib
+import inspect
+
+import six
+from stix2patterns.grammars.STIXPatternLexer import STIXPatternLexer
+from stix2patterns.grammars.STIXPatternParser import (STIXPatternParser,
+                                                      TerminalNode)
+from stix2patterns.grammars.STIXPatternVisitor import STIXPatternVisitor
+from stix2patterns.validator import STIXPatternErrorListener
+
+from antlr4 import CommonTokenStream, InputStream
+from stix2.patterns import _BooleanExpression
+
+# need to import all classes because we need to access them via globals()
+from .patterns import (BinaryConstant, BooleanConstant, FloatConstant,
+                       FollowedByObservationExpression, HexConstant,
+                       IntegerConstant, RepeatQualifier, StartStopQualifier,
+                       StringConstant, TimestampConstant, WithinQualifier)
+from .patterns import *  # noqa
```

Comment:
Do we need both import statements `from .patterns import *`  and `from .patterns import (...)`?